lib/utils/uid.js

```diff
@@ -4,8 +4,7 @@ let counter = 0
  * @link https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L16143
  * @param {string} prefix
  */
-export default function uid(prefix = "") {
+export default function uid(prefix = "bv_") {
     const id = ++counter
```

Comment:
Since element IDs are a prop, and uses doesn't supply an ID and we try to generate one (i.e. similar to your uid function), the auto-id provided by the SSR rendering will most likely not match with the ID value generated by the client code, and they conflict, so the client bales rehydration of the server rendered markup and blows it away and regenerates it using the client generated ID.  so one different "default" value for a prop (between server and cli3ent) can cause the component (or evn the app) to re-render client side.